src/engine/PiecewiseLinearConstraint.h

```diff
@@ -582,16 +588,19 @@ class PiecewiseLinearConstraint : public ITableau::VariableWatcher
     /**********************************************************************/
     inline bool existsAssignment( unsigned variable ) const
     {
-        return ( _gurobi == nullptr ) ?
-            _assignment.exists( variable )
-            : _gurobi->existsAssignment( Stringf( "x%u", variable ) );
+        if ( _gurobi == nullptr && _tableau != nullptr )
```

Comment:
This is intended to give preference to Gurobi, right? So lets reorder:

if ( gurobi exists )
   return gurobi value;

if ( tableu exists )
   return tableu value

fail

Reply:
Done. Yeah this reads much better..